projects/desyn/index.js

```diff
@@ -33,7 +43,19 @@ Object.keys(config).forEach(chain => {
 
       const pools = logs.map(i=>i.pool)
       const tokens = await api.multiCall({  abi: 'address[]:getCurrentTokens', calls: pools})
-      return api.sumTokens({ ownerTokens: tokens.map((tokens, i) => [tokens, pools[i]])})
+      const calls = []
```

Comment:
again, are the tokens moving out of the contract here? right now, they remain in the contract, so why this change?

Reply:
Previously it was calculated via balanceof, now it is via the contract's getBalance

Reply:
So for the time being this 0x1982b2F5814301d4e9a8b0201555376e62F82428token is not used as a tvl reference, any more questions here?